exporters/geneva/example/foo_library.h

```diff
@@ -0,0 +1,15 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#ifndef ENABLE_METRICS_PREVIEW
+#  include <string>
+
+class foo_library
```

Comment:
nit: make the name of class name consistent, like `FooLibrary` here?

Reply:
yeah good point, will fix this.

Reply:
This is done now.